javascript/node/selenium-webdriver/BUILD.bazel

```diff
@@ -88,3 +88,25 @@ genrule(
     ],
     cmd = "cp $(locations //:license) $(@D)",
 )
+
+nodejs_binary(
```

Comment:
We should delete the json files and then copy them into the right place at build time. You can see how I did it in the python bazel code.